pages/common/sails.md

```diff
@@ -0,0 +1,23 @@
+sails
```

Comment:
Almost missed this one. Please add a header tag, as in `# sails`. I'll be happy to merge then.